frame/support/Cargo.toml

```diff
@@ -35,10 +35,10 @@ paste = "1.0"
 sp-state-machine = { version = "0.28.0", default-features = false, optional = true, path = "../../primitives/state-machine" }
 bitflags = "1.3"
 impl-trait-for-tuples = "0.2.2"
-smallvec = "1.11.0"
```

Comment:
Does this also have to be reverted?  
Otherwise please undo the reversions in the `frame` folder.